cylc/flow/cfgspec/globalcfg.py

```diff
@@ -984,7 +984,9 @@ def default_for(
 
             .. versionadded:: 8.0.0
         """):
-            with Conf('<install target>'):
+            with Conf('<install target>', desc="""
+                Install target where directories to be symlinked are located.
```

Comment:
Symlinking is optional, and file installation is the important thing?

```suggestion
                Where to install workflow files. A platform with a shared filesystem
                needs only one install target.
```

Reply:
Oops, sorry, I just saw `install target` and didn't check the context!!

Reply:
Although, we probably should refer to the platforms docs on what an install target is, or explain it here like I did. (The item name is already "install target", so the value obviously must be a particular "install target" ... in which case we should either link or explain what that means).